plugin/pkg/scheduler/generic_scheduler.go

```diff
@@ -166,20 +176,37 @@ func PrioritizeNodes(pod *api.Pod, podLister algorithm.PodLister, priorityConfig
 		priorityFunc := priorityConfig.Function
 		prioritizedList, err := priorityFunc(pod, podLister, minionLister)
 		if err != nil {
-			return algorithm.HostPriorityList{}, err
+			return schedulerapi.HostPriorityList{}, err
 		}
 		for _, hostEntry := range prioritizedList {
 			combinedScores[hostEntry.Host] += hostEntry.Score * weight
 		}
 	}
+	if len(extenders) != 0 && minionLister != nil {
+		nodes, err := minionLister.List()
+		if err != nil {
+			return schedulerapi.HostPriorityList{}, err
+		}
+
+		for _, extender := range extenders {
+			prioritizedList, err := extender.Prioritize(pod, &nodes)
```

Comment:
priorityFunc() (L177) gets passed a podLister which allows the priority function to list all pods. To make the extender priority function interface as similar to the regular priority function interface as possible, I think you should pass all pods here, not just the pod being scheduled.